code/__DEFINES/bitrunning.dm

```diff
@@ -26,3 +26,22 @@
 
 /// Camera network bitrunner bodycams are on
 #define BITRUNNER_CAMERA_NET "bitrunner"
+
+/**
+ * Bitrunner Domain External Load Restriction Bitflags
+ */
+/// Domain forbids external sources from loading items onto avatars
+#define DOMAIN_FORBIDS_ITEMS (1<<0)
+/// Domain forbids external sources from loading abilities onto avatars
+#define DOMAIN_FORBIDS_ABILITIES (1<<1)
+
+/// Combination flag for blocking anything from being loaded onto avatars by external sources
+#define DOMAIN_FORBIDS_ALL (DOMAIN_FORBIDS_ITEMS | DOMAIN_FORBIDS_ABILITIES)
```

Comment:
why not just use `ALL`? are these flags not all going to be restrictions?

Reply:
good thought, will do
thought for a bit about whether to remove it entirely or to define it as `ALL`, but something about the latter felt saner to me personally, so going with that